### PR TITLE
Add keystone service #8

### PR DESCRIPTION
**Pull request by loles**

### CODE REVIEW

File: composer_files/openstack.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+resources:
```
Comment (dshulyak) [inline composer_files/openstack.yaml]: it seems that there is no connection between keystone and mariadb, maybe there should be at least event?

File: composer_files/openstack.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        db_user: root
+        db_password: mariadb
+        db_name: keystone
+        db_host: mariadb.default.svc.cluster.local
+        debug: True
+        log_file: '/var/log/keystone/keystone.log'
+      hash:
+        computable:
+          connections:
+          - keystone_config_map::C::NO_EVENTS
```
Comment (dshulyak) [inline composer_files/openstack.yaml]: results in - Exception: Trying to connect value-.* to itself.

File: composer_files/openstack.yaml (hunk 1 above)
Comment (loles) [inline composer_files/openstack.yaml]: Yes, I will add event

File: composer_files/keystone.yaml [hunk 3]
```diff
@@ -1,30 +1,108 @@
...
+          - keystone_config_map::admin_token::NO_EVENTS
 
   - id: keystone_api_deployment
     from: k8s/KeystoneDeployment
     input:
       dns_name: keystone-api-deployment
       labels:
         app: keystone-api
-      command: /usr/bin/keystone-all
+      command: ["/usr/bin/bash", "-c", "keystone-manage db_sync && /usr/bin/keystone-all"]
```
Comment (dshulyak) [inline composer_files/keystone.yaml]: will it work with >1 keystone?

Comment (loles) [inline composer_files/keystone.yaml]: As long as it doesn't start all at the same time it should. 
It's just workaround until I figure out how to do it differently

**Pull request merged**